### ISSUE TITLE
Way to copy .js files without allowJs flag

### ISSUE THREAD

**Issue by npdev453** [swimauger/tsc-hooks#25]
Hi everyone, I'm looking for a way to copy .js files in project without `allowJs` using this library. 

Because in `composition` mode with `references` and  with `allowJ` options, `tsc`  trying to generate declarations for all `.js` files (or they should have own .d.ts) and it's a problem for a huge project. 

I see that all `.js` files skipped there, but maybe we can make it more responsive and allow copying `.js` by special flag also?

**Comment by swimauger** [comment]
Hi @npdev453,

I'm a little confused... Are you saying the `allowJs` option doesn't work with the other options you are using?

**Comment by npdev453** [comment]
Yes, and I can't enable this flag, because it uses not for only copying `.js`s files. 
But I still need to copy .js files into `outDir` and don't want to use any cmd or bash commands.

**Comment by swimauger** [comment]
Why not use `allowJs` with the exclude property? But also, `allowJs` shouldn't be copying over anything, but .js or .jsx files
Re: https://www.typescriptlang.org/tsconfig#include

**Comment by npdev453** [comment]
Heh, yes, `allowJs: true` do not copy other files, but it does some another things ⚠️  not **only** copying.

https://www.typescriptlang.org/tsconfig#allowJs

`.js`s files going to be a part of project and trying to be proceeded with compiler (especially to generate declarations when `declaration: true`) 

When I set `allowJs` and 'exclude: [`*.js`]', then nobody going to copy `.js` at all.
